styles.css

```diff
@@ -9,6 +9,7 @@ body{
 }
 header, footer, aside, .main{
   padding: 1em;
+  margin: 0 auto;
```

Comment:
You wanted to set `margin: 0` for the direct child of the above selectors. In the case of `header`, you would do something like this...

```css
header > * {
  margin: 0;
}